help/en/html/tools/logixng/reference/chapter5.shtml

```diff
@@ -1152,6 +1152,34 @@
         "images/chapter5/digital_action_display.png" alt="Chapter 5 digital action display list"></a>
       </div>
 
+      <div style="margin-left: 2em;">
+        <a href="images/chapter5/digital_action_displayold.png"><img class="image-border" src=
+        "images/chapter5/digital_action_displayold.png" alt="Chapter 5 digital action display list"></a>
+      </div>
```

Comment:
This `div` seems to be a duplicate of the one before, except that the file name is `digital_action_displayold.png`. Copy/paste error?